core/src/components/badge/badge.common.scss

```diff
@@ -1,3 +1,4 @@
+@use "../../themes/functions.color" as color;
```

Comment:
I went with this approach since the current "globals" [usage](https://github.com/ionic-team/ionic-framework/blob/4874e61663f210cace81b414254d269fbbfac532/core/src/components/avatar/avatar.ionic.scss#L1) is being brought in via the `ionic` file. Since this file is a common file, we don't want any themes influencing it. Hence the reason why I didn't import it via the `ionic` or `native` files.